### PR TITLE
Update better_zip.py

### PR DESCRIPTION
**Pull request by ghost**
Bugfixing name => cast to str all elements if exists.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
The same question as the other PR.

**Comment by ghost** [comment]
@pedrobaeza : Same answer as the other PR.

File: base_location/better_zip.py [hunk 1]
```diff
@@ -51,6 +52,7 @@ def name_get(self, cursor, uid, ids, context=None):
                 name.append(bzip.state_id.name)
             if bzip.country_id:
                 name.append(bzip.country_id.name)
+            name = [str(x) for x in name if x]
```
Comment (bwrsandman) [inline base_location/better_zip.py]: It would be better to use ustr from openerp.tools

Comment (ghost) [inline base_location/better_zip.py]: @bwrsandman 
Hello, I did another commit with this suggestion, 

thank you!

**Comment by pedrobaeza** [comment]
As said in #58, this doesn't apply, so I close it.

**Pull request closed without merging**